<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBSystemInfoPageUi</class>
 <widget class="QFrame" name="OpenRGBSystemInfoPageUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="SMBusDataText"/>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="DumpAddressBox">
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string>0x</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="displayIntegerBase">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="DumpButton">
     <property name="text">
      <string>Dump Device</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="DumpAddressLabel">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="SMBusDumpLabel">
     <property name="text">
      <string>SMBus Dumper:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="SMBusAdaptersLabel">
     <property name="text">
      <string>SMBus Adapters:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="DetectButton">
     <property name="text">
      <string>Detect Devices</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="SMBusAdaptersBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="SMBusDetectorLabel">
     <property name="text">
      <string>SMBus Detector:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="SMBusDetectModeLabel">
     <property name="text">
      <string>Detection Mode:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="SMBusDetectionModeBox"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SMBusAdaptersBox</tabstop>
  <tabstop>SMBusDetectionModeBox</tabstop>
  <tabstop>DetectButton</tabstop>
  <tabstop>DumpAddressBox</tabstop>
  <tabstop>DumpButton</tabstop>
  <tabstop>SMBusDataText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
